core/controllers/tasks.py

```diff
@@ -79,7 +80,7 @@ def post(self):
         exploration_rights = (
             rights_manager.get_exploration_rights(exploration_id))
         exploration = exp_services.get_exploration_by_id(exploration_id)
-        suggestion = feedback_services.get_suggestion(thread_id)
+        suggestion = suggestion_services.get_suggestion_by_id(thread_id)
```

Comment:
Hey @seanlip I just realized my branch is called develop. Will this be problematic? I can create a new pull request with a corrected branch name if needed.

Reply:
I think it might mess up your develop branch on your fork, yeah. Would you be OK with doing a new PR to keep everything clean (and to get used to the standard dev workflow)?

Thanks!

Reply:
Yes, I'll do that now. Sorry about that! I'm new to opensource and am learning the rules as I go.

Reply:
Done.